public/resources/core/js/api/party/individual.controller.js

```diff
@@ -0,0 +1,211 @@
+/**
+ * @author Francisco de la Vega <fdelavega@conwet.com>
+ *         Jaime Pajuelo <jpajuelo@conwet.com>
+ *         Aitor Magán <amagan@conwet.com>
+ */
+
+
+(function () {
+
+    'use strict';
+
+    angular
+        .module('app')
+        .controller('IndividualUpdateCtrl', IndividualUpdateController)
+        .controller('ContactMediumUpdateCtrl', ContactMediumUpdateController);
+
+    function IndividualUpdateController($state, $scope, $rootScope, EVENTS, COUNTRIES, Individual) {
+        /* jshint validthis: true */
+        var vm = this;
+
+        $scope.COUNTRIES = COUNTRIES;
+
+        var emailAddress = {
+            emailAddress: ""
+        };
+
+        var telephoneNumber = {
+            type: "",
+            number: ""
+        };
+
+        var postalAddress = {
+            streetOne: "",
+            postcode: "",
+            city: "",
+            country: "",
+            stateOrProvince: ""
+        };
+
+        vm.MEDIUM_TYPES = Individual.MEDIUM_TYPES;
+
+        vm.contactMedium = {
+            preferred: false,
+            type: vm.MEDIUM_TYPES.EMAIL_ADDRESS,
+            medium: angular.copy(emailAddress)
+        };
+        vm.contactMediumSelected = [];
+
+        var STATUS = {
+            NOT_FOUND: 404
+        };
+
+        Individual.detail().then(function (individualRetrieved) {
+            retrieveIndividual(individualRetrieved);
+        }, function (response) {
+            if (response.status === STATUS.NOT_FOUND) {
+                retrieveIndividual();
+            } else {
+                $rootScope.$broadcast(EVENTS.MESSAGE_ADDED, 'error', {
+                    error: Utils.parseError(response, "Unexpected error trying to retrieve your personal information.")
```

Comment:
Please, be coherent: use always single quotes or double quotes (single quotes are preferred as they have been used along all the project).